ansible/playbook/tasks/update.yml

```diff
@@ -160,3 +160,11 @@
 
     - name: Check supervisord log
       debug: var=maintail_result.stdout_lines
+
+    # New config will be applied after next reboot.
```

Comment:
I don't reload systemd configs because it will restart supervisord which might stop pmm-update(not 100% sure).